<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gonnengjiemian</class>
 <widget class="QWidget" name="gonnengjiemian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>771</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background:rgb(255, 255, 255);
	font-size:20px;
	border: none;border-radius:20px;
	
}

Qlabel{
font-color:208,208,208;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_ip">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>160</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color:208,208,208;
	background-color:#FFFFFF;
	font-size:15px;
	border-style:outset;
	border:2px solid rgb(20,196,188);
	border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>1.14.17.13</string>
    </property>
    <property name="placeholderText">
     <string>IP地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>9</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>14</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 14pt '微软雅黑 Light';color: rgb(123,115,106);</string>
    </property>
    <property name="text">
     <string>客户端</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>210</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color:208,208,208;
	background-color:#FFFFFF;
	font-size:15px;
	border-style:outset;
	border:2px solid rgb(20,196,188);
	border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>5901</string>
    </property>
    <property name="placeholderText">
     <string>端口号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startOrStop">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>260</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background:#14C4BC;
	border-style:outset;
	border-radius:10px;
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(22,218,208);
	
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
    </property>
    <property name="text">
     <string>Link</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background:#14C4BC;
	border-style:outset;
	border-radius:10px;
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(22,218,208);
	
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
    </property>
    <property name="text">
     <string>zaixian</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background:#14C4BC;
	border-style:outset;
	border-radius:10px;
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(22,218,208);
	
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
    </property>
    <property name="text">
     <string>shebei2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background:#14C4BC;
	border-style:outset;
	border-radius:10px;
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(22,218,208);
	
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
    </property>
    <property name="text">
     <string>shebei1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>100</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>风扇状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>设备状态</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background:#14C4BC;
	border-style:outset;
	border-radius:10px;
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(22,218,208);
	
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
    </property>
    <property name="text">
     <string>关风扇</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background:#14C4BC;
	border-style:outset;
	border-radius:10px;
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(22,218,208);
	
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
    </property>
    <property name="text">
     <string>开风扇</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_ip</tabstop>
  <tabstop>lineEdit_port</tabstop>
  <tabstop>pushButton_startOrStop</tabstop>
  <tabstop>pushButton_send_2</tabstop>
  <tabstop>pushButton_send_5</tabstop>
  <tabstop>pushButton_send_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
